diffblue.yml

```diff
@@ -5,20 +5,61 @@ phases:
  timeout: 300
  cbmcArguments:
    classpath: '/tools/cbmc/models-simple-overlay.jar:/tools/cbmc/models.jar:.'
-   max-nondet-array-length: 100
-   java-max-vla-length: false
+   java-assume-inputs-non-null: true
+   java-max-vla-length: 48
+   max-nondet-array-length: 10
+   max-nondet-string-length: 10
+   string-printable: true
+   throw-runtime-exceptions: false
    unwind: 1
+ nextPhase:
+   not_analyzed: null
+   time_out: 3
 -
  timeout: 300
  cbmcArguments:
    classpath: '/tools/cbmc/models-simple-overlay.jar:/tools/cbmc/models.jar:.'
-   max-nondet-array-length: 100
-   java-max-vla-length: false
+   java-assume-inputs-non-null: false
+   java-max-vla-length:  96
+   max-nondet-array-length: 20
+   max-nondet-string-length: 50
+   string-printable: true
+   throw-runtime-exceptions: false
    unwind: 2
+ nextPhase:
+   time_out: 3
 -
  timeout: 300
  cbmcArguments:
    classpath: '/tools/cbmc/models.jar:.'
-   max-nondet-array-length: 100
-   java-max-vla-length: false
+   java-assume-inputs-non-null: false
+   java-max-vla-length:  192
+   max-nondet-array-length: 30
+   max-nondet-string-length: 10
+   string-printable: false
+   throw-runtime-exceptions: true
    unwind: 3
+-
+ timeout: 300
```

Comment:
:grey_question: Would it be worth splitting this one into two phases, one with lower values and `throw-runtime-exceptions: false` and a second one with higher values and `throw-runtime-exceptions: true`? Assuming that many executions will get 100% coverage in the first run this might save us some time. Same for the last phase.

Reply:
Hm, maybe.. I'll try and run the benchmark with the setup where as fallback we would have copy of first phase with `load-containing-class-only: true`, then copy of third phase with `load-containing-class-only: true`, and then the same for `single-function-only: true`. I'm curious if and how much more time it will take.. and coverage of course.

Reply:
That sounds like a good idea, let me know what the result was and when I should re-review this PR. I'm happy with anything beyond the current discussions on this PR to be left for fine-tuning later. :slightly_smiling_face: